docker/run_ftpscraper_wrapper.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Checks the cached ftpscraper log and if it's less than 7 days old, replays it
+# which is super duper fast.
+
+set -eo pipefail
+
+# Fetch and update release information for these products (comma-delimited)
+PRODUCTS="firefox,mobile"
+CRONTABBERCMD="./socorro-cmd crontabber"
+CACHEDIR="./.cache"
```

Comment:
Not sure if a line similar to: `# The assumption is that you're running this from /app inside the container` could still be useful?

Reply:
The problem is that `stat` on osx doesn't support the -c flag. I'll rework how this works.